http/rest-client/src/test/java/io/quarkus/ts/http/restclient/GzipMaxInputIT.java

```diff
@@ -0,0 +1,76 @@
+package io.quarkus.ts.http.restclient;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.zip.GZIPOutputStream;
+
+import jakarta.ws.rs.core.MediaType;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class GzipMaxInputIT {
+
+    final byte[] gzip_bellow_max_input_1K = new byte[1000];
+    final byte[] gzip_max_input_10M = new byte[10000000];
```

Comment:
I would suggest to also add a test, which uses chunk a little bit bigger, than the limit.

Reply:
done